graphqlapi/src/main/java/io/stargate/graphql/schema/fetchers/CassandraFetcher.java

```diff
@@ -25,8 +31,39 @@ public final ResultT get(DataFetchingEnvironment environment) throws Exception {
 
     String token = httpAwareContext.getAuthToken();
     StoredCredentials storedCredentials = authenticationService.validateToken(token);
-    DataStore dataStore = DataStore.create(persistence, storedCredentials.getRoleName());
 
+    Parameters parameters;
+    Map<String, Object> options = environment.getArgument("options");
+    if (options != null) {
+      ImmutableParameters.Builder builder = Parameters.builder();
+
+      Object consistency = options.get("consistency");
+      if (consistency != null) {
+        builder.consistencyLevel(ConsistencyLevel.valueOf((String) consistency));
+      }
+
+      Object serialConsistency = options.get("serialConsistency");
+      if (serialConsistency != null) {
+        builder.serialConsistencyLevel(ConsistencyLevel.valueOf((String) serialConsistency));
+      }
+
+      Object pageSize = options.get("pageSize");
+      if (pageSize != null) {
```

Comment:
shouldn't we also that is instanceof Integer?
` if (pageSize != null && pageSize instanceof Integer)`
(similar for other checks)?

Reply:
The GraphQL parser will make sure the correct type is provided, so there's no need for additional validation.